GetStream/stream-chat-android
# Fix not equal name filter

### 🎯 Goal

Fix "operator $ne is not supported for custom fields" error when querying channels in new apps

### 🛠 Implementation details

$ne operator shouldn't be used for custom fields and it was deprecated in new apps.

### 🧪 Testing

1. Create a new app in the dashboard
2. Check create channel screen
3. Verify the list of users loads without errors

### ☑️ Checklist
